### PR TITLE
954 bootstrap current fraction scaling formula for steady state machines

### PR DESCRIPTION
**Pull request by chris-ashe**
## Description

A new model by Sakai et al has been implemented
image

This should be applicable for all aspect ratios.

Unit tests created for large tokamak and ST case

Sakai et al. says at the start of the paper that the dataset used to fit the model includes the toroidal diamagnetic current. Thus we remove the diamagnetic fraction from the calculated value when the function is called 

## Checklist

I confirm that I have completed the following checks:

- [x] I have justified any large differences in the regression tests caused by this pull request in the comments.
- [x] I have added new tests where appropriate for the changes I have made.
- [x] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
- [x] If I have made documentation changes, I have checked they render correctly.
- [x] I have added documentation for my change, if appropriate.

### CODE REVIEW

**Review by stuartmuldrew — CHANGES_REQUESTED** [review]
Minor comment about diamagnetic current options

File: documentation/proc-pages/physics-models/plasma_current.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ existence of pedestals, whereas the Sauter et al. scaling
 | 2 | General scaling -- To use a more general scaling method, set `bscfmax` to the maximum required bootstrap current fraction ($\leq 1$).
 | 3 | Numerically fitted scaling [^8] -- To use a numerically fitted scaling method, valid for all aspect ratios, set `bscfmax` to the maximum required bootstrap current fraction ($\leq 1$).
 | 4 | Sauter, Angioni and Lin-Liu scaling [^9] [^10] -- Set `bscfmax` to the maximum required bootstrap current fraction ($\leq 1$).
+| 5 | Sakai, Fujita and Okamoto scaling [^11] -- Set `bscfmax` to the maximum required bootstrap current fraction ($\leq 1$).
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/physics-models/plasma_current.md]: The warning about taking the calculated diamagnetic fraction from the calculated bootstrap value has been added

**Comment by mkovari** [comment]
Thanks - a helpful piece of work.
Does it change the results significantly?

**Review by stuartmuldrew — APPROVED** [review]
Happy with error message addition.

**Pull request merged**